src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -1174,17 +1174,20 @@ protected override SyntaxNode GetSymbolDeclarationSyntax(SyntaxReference referen
         {
             isAmbiguous = false;
 
-            if (node.IsKind(SyntaxKind.Parameter, SyntaxKind.TypeParameter, SyntaxKind.UsingDirective, SyntaxKind.NamespaceDeclaration))
-            {
-                return null;
-            }
-
             if (editKind == EditKind.Update)
             {
-                if (node.IsKind(SyntaxKind.EnumDeclaration))
+                if (node.IsKind(SyntaxKind.Parameter, SyntaxKind.TypeParameter))
                 {
-                    // Enum declaration update that removes/adds a trailing comma.
```

Comment:
> Enum declaration update that removes/adds a trailing comma. [](http://example.com/codeflow?start=22&length=60)

How do we handle "Enum declaration update that removes/adds a trailing comma." now?

Reply:
No change (there are tests covering it that still pass), the edit isn't reported because the symbol is an INamedTypeSymbol so its just processed for attributes.